src/main/java/redis/clients/jedis/CommandObjects.java

```diff
@@ -809,6 +817,15 @@ protected CommandArguments commandArguments(ProtocolCommand command) {
     return new CommandObject<>(commandArguments(BLMOVE).blocking().key(srcKey)
         .key(dstKey).add(from).add(to).add(timeout), BuilderFactory.BINARY);
   }
+
+  public final CommandObject<Map<byte[], List<byte[]>>> lmpop(ListDirection from, byte[]... keys) {
+    return new CommandObject<>(commandArguments(LMPOP).add(keys.length).keys((Object[]) keys).add(from), BuilderFactory.GROUPS_LIST_BINARY);
+  }
+
+  public final CommandObject<Map<byte[], List<byte[]>>> lmpop(ListDirection from, int count, byte[]... keys) {
+    return new CommandObject<>(commandArguments(LMPOP).add(keys.length).keys((Object[]) keys).add(from).add(COUNT).add(count), BuilderFactory.GROUPS_LIST_BINARY);
+  }
```

Comment:
If we agree on [1], we can return `List<Object>` here.

Reply:
why not `List<KeyedListElements>` ?

Reply:
> @zeekling To leave a variant so that users can make their own implementation.

ok, i see.

Reply:
i modified it @sazzad16